autoBugTracker.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import executeUserScript
+
+
+class autoBugTracker(object):
+    def __init__(self):
+        print("***Auto Bug Tracker***\n")
+        self.execute = executeUserScript.executeUserScript()
+
+
+    def parsingCommandLineArguments(self):
+        '''' Return dictionary
```

Comment:
Please follow the same commenting style like the rest of the project

Reply:
By Product owner follow the comments style of Python 3.8. Document.

Reply:
I thought 3.8 was with double quotes instead of single quotes?